src/config_paths.rs

```diff
@@ -49,3 +49,16 @@ pub fn expand(config_paths: Vec<PathBuf>) -> Option<Vec<PathBuf>> {
     }
     Some(paths)
 }
+fn default_config_paths() -> Vec<PathBuf> {
+    let config_paths = if cfg!(unix) {
+        DEFAULT_UNIX_CONFIG_PATHS.clone()
+    } else if cfg!(windows) {
+        let program_files = env!("ProgramFiles");
```

Comment:
Maybe this would be more understandable if it was built in https://github.com/timberio/vector/blob/64b0754b34e08b4449270e510262040341a4ea12/src/config_paths.rs#L7 ?

https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=2966056be5134e91cb37089dbfe301b5

Reply:
Yes, this is better !

I also need to target the right recommended default configuration folder. I guess I'll follow what you proposed here ? https://github.com/timberio/vector/issues/1738#issuecomment-583493898

Maybe `%APPDATA%` ?